source/dlangbot/app.d

```diff
@@ -334,6 +338,10 @@ void handlePR(string action, PullRequest* _pr)
 
         auto oldComments = getBugComments(refs.map!(r => r.id).array);
 
+        // Update scoreboard if any points are obtained
+        if (points)
+            updateScoreBoard(pr.head.user.get().login, points);
```

Comment:
As I suggested elsewhere, instead of modifying the points directly, it's better to store the event in the DB, so that we have a full history of the actions that the use took. That way the gamification logic is decoupled from the actual data and can be modified independently. (It is very unlikely that we're going to come up with the perfect formula on our first try.)